### ISSUE TITLE
Cruel Ultimatum Bug?

### ISSUE THREAD

**Issue by radiasian** [magefree/mage#9127]
Cruel Ultimatum seems to only do the effects on the opponent, not on the player
![image](https://user-images.githubusercontent.com/101856030/174462973-5c9aec2b-9e98-4d58-a03b-4b08d943c2ee.png)

**Comment by Susucre** [comment]
Indeed, there is a fail in the implemented logic: if you can't return a creature into your hand, there is no lifegain nor card draw.

**Comment by jeffwadsworth** [comment]
Weird that this bug wasn't noted before.  Card is ancient.
